internal/httputil/constants.go

```diff
@@ -7,3 +7,11 @@ const (
 	// Especially useful when working with single page apps (SPA).
 	HeaderPomeriumResponse = "x-pomerium-intercepted-response"
 )
+
+// HeadersContentSecurityPolicy are the content security headers added to the service's handlers
+// by default includes profile photo exceptions for supported identity providers.
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src
+var HeadersContentSecurityPolicy = map[string]string{
+	"Content-Security-Policy": "default-src 'none'; style-src 'self'; img-src 'self' https://*.googleusercontent.com;",
```

Comment:
> https://*.googleusercontent.com

To allow user photos to be remotely loaded on the profile page we either need to:

1) Add each identity provider, as done here.
2) Make `img-src` less strict. The concern is that a potentially hacked JWT/idp provider could serve the user with a malformed `profile_picture` claim which would allow the owner of that remote domain to perform a `xss` / injection attack. 

I'm somewhat comfortable going for 2 because if an attacker is able to forge our signed cookies, or hack the trusted identity provider, we have much bigger issues than a `xss`.

Reply:
Done.